install/parse-cli.sh

```diff
@@ -45,6 +45,7 @@ depwarn() {
 if [ ! -z "${SKIP_USER_PROMPT:-}" ]; then
   depwarn "SKIP_USER_PROMPT variable" "SKIP_USER_CREATION"
   SKIP_USER_CREATION="${SKIP_USER_PROMPT}"
+  APPLY_AUTOMATIC_CONFIG_UPDATES="${SKIP_USER_PROMPT}"
```

Comment:
Slightly nervous about this.

Reply:
Because we will be trying the auto migration/config update without asking about it. Ideally this would be no action on non-interactive installs that said since you try to ensure it is unchanged, I'm fine. Just a little bit nervous in case we missed something.